.github/workflows/standard_spa_pr.yml

```diff
@@ -25,45 +46,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  cache-yarn:
-    name: Cache Yarn
-    runs-on: jupiterone-dev
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Cache yarn dependencies
-        uses: actions/cache@v2
-        id: yarn-cache
-        with:
-          path: |
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
-            **/node_modules
-          key: cache-yarn-${{ hashFiles('**/yarn.lock') }}
-      - name: Install Dependencies
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        # Eventually we want to flip to npm and use `npm ci` here
-        run: |
-          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_AUTH_TOKEN }}" > .npmrc
-          yarn --frozen-lockfile
-
   validate:
     name: Validate
-    runs-on: jupiterone-dev
-    needs: cache-yarn
+    runs-on: ${{ inputs.runs_on }}
+    timeout-minutes: 15
+    if: ${{ inputs.use_validate }}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Fetch Yarn Cache
-        uses: actions/cache@v2
-        id: yarn-cache
+      - uses: actions/checkout@v3
+      # This should be part of the runner
+      ## Install yarn START
```

Comment:
This is annoying but only takes an extra 6 seconds in the build. When our in-house runners have yarn@1 installed by default we can disable this and then remove.